WooCommerce/Classes/ViewRelated/Products/Edit Product/Inventory Settings/ProductInventorySettingsViewController.swift

```diff
@@ -29,7 +29,7 @@ final class ProductInventorySettingsViewController: UIViewController {
 
     private var skuBarcodeScannerCoordinator: ProductSKUBarcodeScannerCoordinator?
 
-    private var cancellable: ObservationToken?
+    private var cancellable: AnyCancellable?
```

Comment:
Nit: we should rename this to `sectionsSubscription` to make it clearer I think.

Reply:
Sorry I forgot to rename everything since there were changes in many places, and renaming in Xcode was slow for me (even though they were just local variables 🤦🏻‍♀️). Updated in https://github.com/woocommerce/woocommerce-ios/pull/5989/commits/97e9eba8ec2623c15ffaf10dd7f746b401aa4cc7